store/postgres/src/relational_queries.rs

```diff
@@ -539,7 +539,14 @@ impl EntityData {
                     // table column; those will be things like the
                     // block_range that `select *` pulls in but that we
                     // don't care about here
-                    if let Some(column) = table.column(&SqlName::verbatim(key)) {
+                    if key == "vid" {
```

Comment:
There's a lot of literal `"vid"` strings here, it would be better to use a constant.